src/Corax/Constants.cs

```diff
@@ -14,7 +14,7 @@ public static class Constants
         
         public static readonly string NullValue = Encodings.Utf8.GetString(new []{(byte)0});
```

Comment:
Can you use `NullValueSpan` here now?

Reply:
I moved byte array to separate variable instead cause we're reusing it

Reply:
Fixed